api/libproject/devstart.rb

```diff
@@ -88,6 +88,128 @@ def dev_up(*args)
   :fn => lambda { |*args| dev_up(*args) }
 })
 
+def setup_local_environment()
+  root_password = ENV["MYSQL_ROOT_PASSWORD"]
+  ENV.update(Workbench::read_vars_file("db/vars.env"))
+  ENV["DB_HOST"] = "127.0.0.1"
+  ENV["MYSQL_ROOT_PASSWORD"] = root_password
+  ENV["DB_CONNECTION_STRING"] = "jdbc:mysql://127.0.0.1/workbench?useSSL=false"
+  ENV["PUBLIC_DB_CONNECTION_STRING"] = "jdbc:mysql://127.0.0.1/public?useSSL=false"
+end
+
+def run_local_migrations()
+  setup_local_environment
+  # Runs migrations against the local database.
+  common = Common.new
+  Dir.chdir('db') do
+    common.run_inline %W{./run-migrations.sh main}
+    common.run_inline %W{./run-migrations.sh data local}
+  end
+  Dir.chdir('db-cdr') do
+    common.run_inline %W{./generate-cdr/init-new-cdr-db.sh --cdr-db-name cdr}
+    common.run_inline %W{./generate-cdr/init-new-cdr-db.sh --cdr-db-name public}
+  end
+  common.run_inline %W{gradle :tools:loadConfig -Pconfig_file=../config/config_local.json}
+end
+
+Common.register_command({
+  :invocation => "run-local-migrations",
```

Comment:
There's an existing command called `run-local-all-migrations`; this seems to do something pretty different. Perhaps that other command could be parameterized, or else this could be renamed?

Reply:
Sorry, I agree our goal names here are confusing. 

We may at some point want to break up this ginormous file into pieces, which would help. These targets are intended primarily for use by CircleCI (though you can run them locally.) 

I don't have great ideas on other names, let me know if you have suggestions.